std/algorithm/iteration.d

```diff
@@ -2029,9 +2029,11 @@ version(none) // This requires support for non-equivalence relations
 /**
 Lazily joins a range of ranges with a separator. The separator itself
 is a range. If a separator is not provided, then the ranges are
-joined directly without anything in between them (often called $(D flatten)
+joined directly without anything in between them (often called `flatten`
 in other languages).
 
```

Comment:
oops, it seems the other clause i'm talking about does not show above. It is just just below what is shown.